lib/markbind/lib/parser.js

```diff
@@ -438,7 +439,7 @@ Parser.prototype.renderFile = function (file, config) {
       }
       const fileExt = utils.getExtName(file);
       if (fileExt === 'md') {
-        const inputData = md.render(data.toString());
+        const inputData = md.render(fm(data.toString()).body);
```

Comment:
What does this do?

Reply:
This was meant to only render the body of a html file (minus the front matter), but it is not necessary. Let me remove it.